packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -195,6 +195,14 @@ export function createRococo (t: TFunction): EndpointOption {
         providers: {
           SubDAONetwork: 'wss://parachain.subdao.network'
         }
+      },
+      {
+        info: 'rococoTrustBase',
+        paraId: 6633,
+        text: t('rpc.rococo.trustbase', 'TrustBase PC1', { ns: 'apps-config' }),
+        providers: {
+          SubDAONetwork: 'wss://rococo.trustednodes.net'
```

Comment:
Is `SubDAONetwork` network here correct? Or is it rather `'Trusted Nodes'`?

Reply:
thanks